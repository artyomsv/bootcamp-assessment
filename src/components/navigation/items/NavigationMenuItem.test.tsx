import * as React from 'react';
import {createShallow} from '@material-ui/core/test-utils';
import NavigationMenuItem from './NavigationMenuItem.widget';

const shallow = createShallow();

describe('renders without crashing', () => {
  test('if renders', () => {
    const mockFn = jest.fn();
    const result = shallow(<NavigationMenuItem navigateTo={mockFn} title={'Title'} icon={'Icon'}/>);
    expect(result).toMatchSnapshot();
  });
});
