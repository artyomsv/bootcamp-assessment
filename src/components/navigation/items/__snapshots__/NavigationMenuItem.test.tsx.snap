// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing if renders 1`] = `
<NavigationMenuItem
  classes={
    Object {
      "icon": "NavigationMenuItem-icon-3",
      "menuItem": "NavigationMenuItem-menuItem-1",
      "primary": "NavigationMenuItem-primary-2",
    }
  }
  icon="Icon"
  navigateTo={[MockFunction]}
  selected={false}
  title="Title"
/>
`;
